graph/options/options.go

```diff
@@ -146,7 +146,16 @@ func NewOptions(r *http.Request) Options {
 
 	// Process namespaces options:
 	namespaceMap := make(map[string]graph.NamespaceInfo)
-	accessibleNamespaces := getAccessibleNamespaces()
+
+	tokenContext := r.Context().Value("token")
+	var token string
+	if tokenContext != nil {
+		token = tokenContext.(string)
```

Comment:
Can this fail ? Better to protect it with a `[...] token, ok := tokenContext.(string); ok  [...]`